pkg/router/template/template_helper.go

```diff
@@ -55,22 +55,22 @@ func isInteger(s string) bool {
 }
 
 func matchValues(s string, allowedValues ...string) bool {
-	glog.V(5).Infof("matchValues called with %s and %v", s, allowedValues)
+	glog.V(7).Infof("matchValues called with %s and %v", s, allowedValues)
 	for _, value := range allowedValues {
 		if value == s {
-			glog.V(5).Infof("matchValues finds matching string: %s", s)
+			glog.V(7).Infof("matchValues finds matching string: %s", s)
 			return true
 		}
 	}
-	glog.V(5).Infof("matchValues cannot match string: %s", s)
+	glog.V(7).Infof("matchValues cannot match string: %s", s)
```

Comment:
This one should rather be a 5 still. Or even a 4. With allowedValues printed as well.